pages/Configuring/Uncommon-tips-&-tricks.md

```diff
@@ -114,6 +114,62 @@ bind = $mod, S, movetoworkspace, special:magic
 bind = $mod, S, togglespecialworkspace, magic
 ```
 
+## Show desktop
+
+This approach uses same principle as the [Minimize windows using special workspaces](#minimize-windows-using-special-workspaces) section.
+It moves all windows from current workspace to a special workspace named `desktop`.
+Showing desktop state is remembered per workspace.
+
+Create a script:
+
+```sh
+#!/bin/env sh
+
+TMP_FILE="$XDG_RUNTIME_DIR/hyprland-show-desktop"
+
+CURRENT_WORKSPACE=$(hyprctl monitors -j | jq '.[] | .activeWorkspace | .name' | sed 's/"//g')
+
+if [ -s "$TMP_FILE-$CURRENT_WORKSPACE" ]; then
+  readarray -d $'\n' -t ADDRESS_ARRAY <<< $(< "$TMP_FILE-$CURRENT_WORKSPACE")
+
+  for address in "${ADDRESS_ARRAY[@]}"
+  do
+    CMDS+=" dispatch movetoworkspacesilent name:$CURRENT_WORKSPACE,address:$address;"
+  done
+
+  hyprctl --batch$CMDS
+
+  rm "$TMP_FILE-$CURRENT_WORKSPACE"
+else
+  COMMAND="hyprctl clients -j | jq '.[] | select (.workspace .name == \"$CURRENT_WORKSPACE\")'"
+
+  NUMBER_HIDDEN_WINDOWS="$( eval $COMMAND | jq '.address' )"
```

Comment:
is there a reason this is still using eval? Even the jq call could be chained and simplified.